apps/test/unit/code-studio/levels/freeResponse.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import FreeResponse from '@cdo/apps/code-studio/levels/freeResponse';
+import { writeSourceForLevel } from '@cdo/apps/code-studio/clientState';
+
+describe('Free Response', () => {
+  const levelId = 1047;
+  const scriptName = 'test-script';
+  const lastAttemptString = 'This is my final answer';
+  const otherLastAttemptString = 'This is some other answer';
+  let textarea;
+
+  before(() => {
+    textarea = document.createElement('textarea');
```

Comment:
Might add a short comment explaining why we're creating this text area/what it is.